rackcity/api/serializers/user_serializers.py

```diff
@@ -0,0 +1,12 @@
+from rest_auth.registration.serializers import RegisterSerializer
+from rest_framework import serializers
+
+
+class RegisterNameSerializer(RegisterSerializer):
+    first_name = serializers.CharField(required=False)
+    last_name = serializers.CharField(required=False)
+
+    def custom_signup(self, request, user):
+        user.first_name = self.validated_data.get('first_name', '')
+        user.last_name = self.validated_data.get('last_name', '')
+        user.save(update_fields=['first_name', 'last_name'])
```

Comment:
this method is provided by djangorestauth to add additional fields to registration if wanted. it does not override default signup (so username, email, passwords still get updated too)